staging/src/k8s.io/code-generator/cmd/applyconfiguration-gen/args/externaltypes.go

```diff
@@ -114,7 +114,7 @@ func parseExternalMapping(mapping string) (typ types.Name, pkg string, err error
 	}
 	packageTypeStr := parts[0]
 	pkg = parts[1]
-	ptParts := strings.Split(packageTypeStr, ".")
+	ptParts := strings.Split(packageTypeStr, "#")
```

Comment:
Below the comment reads as if `#` was expected?

Reply:
Correct, on line 119, also need to update line 113.

Reply:
Oh wait. To retain compatiblity for existing users,  split on the last `.` here? This which will work regardless of how many dots are in the group name.  `strings.LastIndex` can help with this.

Reply:
I guess the comment below, line 119, still needs to be fixed if someone wants to make a super easy PR.